cli/templates/Python/.dockerignore

```diff
@@ -0,0 +1,13 @@
+# iapp config file may contain secrets
+iapp.config.json
+
+# iapp test directories
+input
+output
+mock
+
+# cache directory
+cache
+
+# python virtual env
+.venv
```

Comment:
our python template does not uses virtual env because dependencies are isolated by the docker image, i added it because venv seems to be a common practice in python development... thought it may be useless